dashboard/app/views/home/audio_test.html.haml

```diff
@@ -0,0 +1,62 @@
+!!! 5
+%html{dir: locale_dir}
+  %head
+    = javascript_include_tag 'soundEffects'
+    :javascript
+      var sounds = new Sounds();
+      sounds.register({
+        id: 'intro',
+        ogg: '/blockly/media/skins/craft/music/vignette4-intro.ogg',
+        mp3: '/blockly/media/skins/craft/music/vignette4-intro.mp3'
+      });
+
+      sounds.register({
+        id: 'success',
+        ogg: '/blockly/media/skins/jigsaw/win.ogg',
+        mp3: '/blockly/media/skins/jigsaw/win.mp3'
+      });
+
+    :css
+      body {
+        text-align: center;
+        background-color: black;
+      }
+      input {
+        padding: 1em;
+        margin: 0.5em;
+        margin-top: 200px;
+        font-size: 24px;
+      }
+      #status {
+        color: white;
+      }
+
+  %body
+    %input{id:'play-now', type:'button', value: 'Play Now'}
+    %input{id:'play-later', type:'button', value: 'Play Later'}
+    %div{id: 'status'}
+      Nothing played yet
+    :javascript
+      var s = document.getElementById('status');
+
+      setTimeout(function () {
+          sounds.whenAudioUnlocked(function () {
+            sounds.play('intro');
+            s.innerHTML = (new Date().getTime()) + ' Starting to autoplay music';
+          });
+        }, 200);
+
+      document.getElementById('play-now').addEventListener('click', function (e) {
+        sounds.play('success');
+        s.innerHTML = (new Date().getTime()) + ' Starting to play sfx';
+      });
+
+      document.getElementById('play-later').addEventListener('click', function (e) {
+        s.innerHTML = (new Date().getTime()) + ' Scheduled music 2s from now...';
+        setTimeout(function () {
+          sounds.play('intro');
+          s.innerHTML = (new Date().getTime()) + ' Starting to play music';
+        }, 2000);
+      });
+
+      console.log('onLoad');
```

Comment:
Done.  This is in the new audio test page which isn't public (for now) but maybe someday it will be.